app/konnect/analytics/index.md

```diff
@@ -1,37 +1,61 @@
-
+| Communicate contextual insights across teams or departments and share important KPIs. | [Custom reports](/konnect/analytics/custom-reports/) |
+| Understand user behavior or pinpoint where issues occur. See detailed information about each individual request made to your APIs. | [Requests](/konnect/analytics/api-requests/)  |
+| Manage Advanced Analytics cost for individual control planes| [Scoping control planes](/konnect/analytics/#ingestion) |
 
 ### Charts and limitations
 
-{{site.konnect_short_name}} charts offer interactive features such as the ability to hover over elements to reveal additional details, and to filter data by selecting items with the chart. {{site.konnect_short_name}} caps the number of entities showing in any activity graph or [custom reports](/konnect/analytics/generate-reports/) to 50. If this limit is exceeded, a warning icon displays at the top of the affected graph or report. 
-
+{{site.konnect_short_name}} charts offer interactive features such as the ability to hover over elements to reveal additional details, and to filter data by selecting items with the chart. {{site.konnect_short_name}} caps the number of entities showing in any activity graph or [custom reports](/konnect/analytics/use-cases/) to 50. If this limit is exceeded, a warning icon displays at the top of the affected graph or report.
 
 ### Team permissions
 
 You can assign {{site.konnect_short_name}} users to specific, predefined Analytic teams. This allows you to give certain users access to only view or manage the Analytics area of your {{site.konnect_short_name}} instance. For more information about the Analytics Admin and Analytics Viewer teams, see the [Teams Reference](/konnect/org-management/teams-and-roles/teams-reference/).
 
+
+### Ingestion
+
+The Advanced Analytics toggle feature allows you to control whether Advanced Analytics data is collected and processed for your API traffic through a control plane.
+This feature provides flexibility in managing your data insights and can help you control costs. There are two states: 
+
+* **On**: When Advanced Analytics is enabled, both basic and advanced data is collected. This gives you valuable insights into API traffic and allows you to generate detailed custom reports.
+* **Off**: When you disable Advanced Analytics, the collection of advanced data stops. However, basic data continues to be collected. This ensures that you still have essential insights and that contextual features in {{site.konnect_short_name}} remain functional.
+
+When Advanced Analytics is set to **off**, you won’t be able to generate [custom reports](/konnect/analytics/use-cases/) but, you will still have access to basic API usage information. 
+
+{:.note}
+> **Transition Time**: It takes a couple of minutes to toggle Advanced Analytics between states. During this time, you will continue to see data being collected.
```

Comment:
```suggestion
> **Transition Time**: When you change the toggle for a control plane, it may take a few minutes for the setting to take effect.  During this time, you may see partial data in Advanced Analytics for your dataplanes.
```